cart.html

```diff
@@ -0,0 +1,187 @@
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FinestStitch</title>
+    <link rel="stylesheet" href="distracted-jones-2f14b5.netlify.app/index.css" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
you can skip to add this font awsome link to every css file by just importing this in your main css file

Reply:
ok